iphone/Maps/UI/PlacePage/Components/PlacePageInfoViewController.swift

```diff
@@ -239,7 +236,8 @@ class PlacePageInfoViewController: UIViewController {
 
   private func addToStack(_ viewController: UIViewController) {
     addChild(viewController)
-    stackView.addArrangedSubview(viewController.view)
+    stackView.addArrangedSubview(viewController.view,
+                                 withSeparator: .default.withInsets(UIEdgeInsets(top: 0, left: 56, bottom: 0, right: 0)))
```

Comment:
How 56 was chosen?

Reply:
56 is used, because every UIImageView in PlacePageInfo use this width and separators was constrained to them.

<img width="1389" alt="SCR-20231005-lvdu" src="https://github.com/organicmaps/organicmaps/assets/79797627/10bf7c30-fe93-4b73-83ff-4c960e0fc488">